runtime/config/manager_internal.go

```diff
@@ -49,21 +49,21 @@ func NewManager(rt *reqtrack.RequestTracker, json jsoniter.API) *Manager {
 	}
 }
 
-func (m *Manager) getComputedCUE(serviceName string) ([]byte, error) {
+func (m *Manager) getComputedCUE(serviceName string) (jsonBytes []byte, found bool, err error) {
```

Comment:
Instead of a separate return, maybe it would be clearer to introduce a specific error type for the case where it's not found:

```go
type NotFoundError struct {
    Service string
    EnvVar string
}

func (e NotFoundError) Error() string {
		return fmt.Sprintf("configuration for service `%s` not found, expected it in environmental variable %s", e.Service, e.EnvVar)
}
```

And then use `errors.As` to check for it

Reply:
Given this is simply for an internal one use function, doesn't it add far more complication to create a new type, and support errors as on it?

Reply:
yeah fair :)